thetis/options.py

```diff
@@ -466,6 +466,8 @@ class CommonModelOptions(FrozenConfigurable):
         None, allow_none=True, help="Source term for 2D continuity equation").tag(config=True)
     tracer_source_2d = FiredrakeScalarExpression(
         None, allow_none=True, help="Source term for 2D tracer equation").tag(config=True)
+    tracer_advective_velocity = FiredrakeVectorExpression(
+        None, allow_none=True, help="Velocity to used in tracer advection equation.").tag(config=True)
```

Comment:
Velocity to be used in tracer advection equation

Reply:
(Correct description)

Reply:
I agree that the description could be better. If I understand correctly, by default the tracer equation uses `uv_2d` field from the shallow water equations. This option allows specifying a custom velocity field instead. This should be made clear to the users here.

Reply:
Also: this option is specific to the 2D model, so I'd put it under `ModelOptions2d` class.

Reply:
I have adapted the definition and changed the option to the ModelOptions2d class. I committed these changes although it doesn't seem to have appeared in the conversation